stackanetes.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: stackanetes
+  name: stackanetes-deployer
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: stackanetes-base
+    #insert a valid image name
+    image: stackanetes-deployer
```

Comment:
build image, push it on quay.io/stackanetes/stackanetes-deployer:0.0.1 and set it as a default value.